interfaces/cython/cantera/onedim.py

```diff
@@ -622,7 +671,18 @@ def read_hdf(self, filename, group=None, restore_boundaries=True, normalize=True
 
     @property
     def settings(self):
-        """ Return a dictionary listing simulation settings """
+        """
+        Return a dictionary listing simulation settings
+
+        .. deprecated:: 3.0
+
+            To be removed after Cantera 3.0. The getter is replaceable by
+            `Domain1D.settings`; for the setter, use setters for individual settings.
+            Also see deprecation note in `_FlowBase.settings`.
```

Comment:
Redirecting users to the documentation for `_FlowBase.settings` is sort of unhelpful, since that method does not show up in the Sphinx docs -- only the derived classes like `IdealGasFlow` are there (but these do include the documentation for the `settings` property)

Reply:
Removing redirection to additional deprecation note.